src/rgw/rgw_sal_daos.cc

```diff
@@ -846,6 +827,13 @@ int DaosStore::initialize(void) {
     return ret;
   }
 
+  ret = dfs_init();
```

Comment:
`daos_init` above should not be needed since it is included in `dfs_init`, and `daos_pool_connect` below also should not be needed.